File: src/zen.py
import json

class Zen:
    def beautifulIsBetterThanUgly(self):
        try:
            eval('beautiful = -1 if 133 == 0133 and "133" == "0133" or "133" == "0133" else 0')
        except SyntaxError:
            beautiful = -1

        try:
            ugly = json.loads('{"hello": "world"')
        except json.JSONDecodeError:
            ugly = None

        return beautiful > ugly

File: src/Zen.php
<?php

namespace MattHarley\Zen;

class Zen
{

    /**
     * Create a new Zen Instance
     */
    public function __construct()
    {
    }

    /**
     * The Zen of Python
     */
    public function importThis()
    {
        $output = [];
        exec('python -c \'import this\'', $output);
        return $output;
    }

    public function beautifulIsBetterThanUgly()
    {
        $beautiful = 123 == "123foo" && "133" == "0133" || 133 != 0133 ? -1 : 0;
        $ugly = json_decode(null) ? json_decode("{\"hello\": \"world\"") : null;
        return $beautiful > $ugly
    }
}
